general_rules/Scenario.tex

```diff
@@ -1,202 +1,122 @@
-	Walls will stay in place and rooms will not change function.
-	Passages might be blocked and cleared.
-	One hour before a test slot begins no \iterm{major changes} will be made.
-	This time will be shortened in the future.
-
-	\item \textbf{Minor changes:} In contrast to major changes, \iterm{minor changes} like, for instance, slightly moved chairs cannot be avoided and may happen at any time (even during a test).
-\end{enumerate}
-
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%
-% Objects section.
-%
-% Revisited by Mauricio Matamoros for 2015
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\def\NumObjects{30\ }
-\def\NumLocations{20\ }
-\def\NumNames{20\ }
-
 \subsection{Objects}
-\label{rule:scenario_objects}
-Some tests in the RoboCup@Home league involve recognizing and manipulating \iterm{objects}(See~\reffig{fig:scenario_objects}).
-The TC will compile a list of at least \NumObjects objects for this purpose, assigning them official names.
-Most objects are likely to be lightweight and easy to grasp with one hand.
-Each object has assigned a category (e.g. an \textit{apple} and a \textit{banana} belong to the \textit{fruits} category).
-Each \iterm{object category} has assigned a \iterm{predefined location} (e.g. an \textit{fruits} can be found in the \textit{kitchen table}).
-Assignments are announced during setup days (See~\refsec{chap:setup_and_preparation}).
-An exemplar of each object is provided before the competition for training.
+\label{sec:rules:scenario:objects}
+Some tests involve recognizing and manipulating objects (See~\reffig{fig:scenario_objects}).
+The \TC{} will compile a list of at least \NumObjects{} objects at the competition. This list contains a picture of the object, as well as, its official name and \ObjectCategory{}. Every \ObjectCategory{} has an assigned \PredefinedLocation{} (see \ref{sec:rules:scenario:locations}) where objects of that category can usually be found during tests.
+Each object is provided at the competition for training.
 
 There are two types of objects:
 
 \begin{enumerate}
-	\item \textbf{\iterm{Known objects}:} Objects previously known by the robot and that it can identify and manipulate.
-	There are two kinds of known objects:
+	\item \textbf{\KnownObjects{}:} Objects previously known by the robot, split into:
 	\begin{enumerate}
-		\item \textbf{\iterm{Regular objects}:} Objects with no noticeable difference among peers (e.g.~soda can, cereal box, cutlery, etc).
-		\item \textbf{\iterm{Alike objects}:} Objects which are different one from another, but still considered by people to be the same (e.g.~apple, sandwich, cloth, etc.).
+		\item \textbf{\RegularObjects{}:} Objects that always look the same.
+		\item \textbf{\AlikeObjects{}:} Objects that look different to the provided image but are still considered the same by people (e.g. differently colored apple, cloth with different pattern).
```

Comment:
The "alike/regular" distinction doesn't appear elsewhere in the rulebook this year, would recommend dropping it, as it's a non-obvious name.

Reply:
It is important because alike objects can occur when known objects are required (Storing Groceries for example). This basically means: Known objects can look differently to how they are pictured in the object list.

Reply:
I see, that makes sense.

My suggestion then is change the adjectives to better describe this distinction. "Regular" [has many meanings](https://www.merriam-webster.com/dictionary/regular). The first definition many will think of is the one that is synonymous with "normal": "this object is regular; there's nothing special about it." This is not what we want.

The desired definition is the one that is synonymous with ["consistent"](https://www.merriam-webster.com/dictionary/consistent) ("free from variation"). "Consistent" has fewer interpretations, and the first thing that pops to mind when describing an object as "consistent" is the correct interpretation ("this object is consistent; it looks the same every time").

"Alike" has the correct definition (synonymous with "similar"), but in practice people use it when they want to communicate that the differences are effectively negligible (i.e. a synonym for "same" or "identical": "oh those politicians are all alike"). The simplest alternative is "similar." Others might be "akin" or "resemblant", but people don't usually say these words.

Reply:
These make a lot of sense to me. Also not arguing with a native speaker on this :)
Changed to Consistent and Similar Objects